console/src/main/webapp/WEB-INF/views/validation.jsp

```diff
@@ -272,11 +272,7 @@ function isNotEmpty(str) {
  * @returns {boolean}
  */
 function isPasswordValid(password) {
```

Comment:
dead code ?

Reply:
replaced original length check by a non-empty check similar to other fields